openwisp_monitoring/device/admin.py

```diff
@@ -63,10 +64,38 @@ class CheckInline(GenericStackedInline):
     fields = ['check_type', 'is_active']
     readonly_fields = ['check_type']
 
+    @property
+    def check_app(self):
+        if 'SAMPLE_APP' in environ:
+            return 'sample_check'
+        return 'check'
+
```

Comment:
This is wrong Aryaman. The application code should not know anything about `SAMPLE_APP`. We use sample app to test the extensibility of OpenWISP modules. Please update your test to check for correct label.

Reply:
Yea, I feel the same this is not a correct way to do that, I'll update it :+1: